libs/groups/src/services/groups-mongo.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { MongoService } from '@backend/mongo';
+import { ConfigService } from '@backend/config';
+import { Group } from '../groups.model';
+
+const collection = 'groups';
```

Comment:
And I guess it's really better to move this Inside of your class and use privately for it unless you really want to use it in another place

Reply:
sure, i should do this in a different pr